source/_integrations/binary_sensor.rest.markdown

```diff
@@ -129,6 +129,11 @@ params:
   description: The query params for the requests.
   required: false
   type: [list, template]
+scan_interval:
+  description: Sets the polling [scan interval](https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval). Default 30 seconds. Additionally, calling service [homeassistant.update](https://www.home-assistant.io/docs/scripts/service-calls/#homeassistant-services) will request an immediate update, rather than waiting for the next scheduled update.
+  required: false
+  type: integer
+  default: 30
```

Comment:
Scan interval is not an property of this integration, it is a property of the platform configuration (not this integration).

Hence not documented here, but here:
<https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval>

Reply:
Fair enough, this may be a little confusing to new users, and I admit, I did not know the scan interval setting was available to me, until I saw it in the rest example.